boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -287,14 +287,15 @@ WinLdrLoadImage(IN PCHAR FileName,
     Status = ArcOpen(FileName, OpenReadOnly, &FileId);
     if (Status != ESUCCESS)
     {
-        // UiMessageBox("Can not open the file.");
```

Comment:
NB: Not sure why `UiMessageBox()` is unwanted here, as there seems to be one in the other cases?

Reply:
It seems to have been commented out on r46017.